d4s/src/main/scala/d4s/models/conditions/Condition.scala

```diff
@@ -100,7 +100,7 @@ object Condition {
     }
   }
 
-  final case class begins_with(path: List[String], prefix: String) extends Condition.Direct {
+  final case class begins_with(path: List[String], prefix: String)(implicit enc: D4SAttributeEncoder[String]) extends Condition.Direct {
```

Comment:
Um, I didn't get it. Why do we need this implicit? We didn't require it previously.

Reply:
@vpodlnk `encodePlain` is using it, https://github.com/PlayQ/d4s/pull/52/files#diff-a270e8652625854ed76c5a35b3997dd7R107; before it always picked up default String encoder, this allows it to pick up and use a local implicit instead if it exists

Reply:
@neko-kai ah, understood, thanks for the explanation!